### PR TITLE
refactor: use link to native changelog

### PR DESCRIPTION
**Pull request by splincode**

### CODE REVIEW

File: projects/demo/src/modules/app/pages.ts [hunk 1]
```diff
@@ -20,7 +20,8 @@ export const pages: TuiDocRoutePages = [
         section: 'Documentation',
         title: 'Changelog',
         keywords: 'release, change, changelog, archive, history',
-        route: DemoRoute.Changelog,
+        route: 'https://github.com/taiga-family/taiga-ui/blob/main/CHANGELOG.md',
```
[the remark this answers is not in the document]
Comment (splincode) [reply projects/demo/src/modules/app/pages.ts:23]: The data will appear automatically after the first major release

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/demo/src/modules/app/pages.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (waterplea) [reply projects/demo/src/modules/app/pages.ts:23]: Links to changelogs of 3 and 2 are broken there

[the remark this answers is not in the document]
Comment (splincode) [reply projects/demo/src/modules/app/pages.ts:23]: Oh, thank you about links

[the remark this answers is not in the document]
Comment (splincode) [reply projects/demo/src/modules/app/pages.ts:23]: fixed

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**